feature-libs/checkout/b2b/root/checkout-b2b-root.module.ts

```diff
@@ -11,7 +12,7 @@ import {
 } from './feature-name';
 
 export function defaultCheckoutComponentsConfig() {
-  const config = {
+  const config: CmsConfig = {
```

Comment:
REMINDER:

we want to put the dependency in here instead of using schematics to generate the 'x'.feature.module for ll.

Reply:
Added a TODO, thanks!